final_task/rss_reader/parser.log

```diff
@@ -0,0 +1,416 @@
+DEBUG    [2019-11-09 16:24:14,829] Links:
+[1]: https://news.google.com/__i/rss/rd/articles/CBMifWh0dHBzOi8vd3d3Lndhc2hpbmd0b25wb3N0LmNvbS9wb2xpdGljcy8yMDE5LzExLzA5L211bHZhbmV5LWFza3Mtam9pbi1sYXdzdWl0LW92ZXItY29uZmxpY3RpbmctZGVtYW5kcy1pbXBlYWNobWVudC10ZXN0aW1vbnkv0gGMAWh0dHBzOi8vd3d3Lndhc2hpbmd0b25wb3N0LmNvbS9wb2xpdGljcy8yMDE5LzExLzA5L211bHZhbmV5LWFza3Mtam9pbi1sYXdzdWl0LW92ZXItY29uZmxpY3RpbmctZGVtYW5kcy1pbXBlYWNobWVudC10ZXN0aW1vbnkvP291dHB1dFR5cGU9YW1w?oc=5(link)
+[2]: 
+
+INFO     [2019-11-09 16:24:42,961] Website is working
+INFO     [2019-11-09 16:24:43,917] Successfull converting into html
+INFO     [2019-11-09 17:47:44,450] Website is working
+DEBUG    [2019-11-09 17:47:52,644] Title: Iran says case open on ex-FBI agent missing there on CIA job
+DEBUG    [2019-11-09 17:47:52,644] Date: Sat, 09 Nov 2019 03:47:16 -0500
+DEBUG    [2019-11-09 17:47:52,644] Link: https://news.yahoo.com/iran-says-case-open-ex-070518919.html
+DEBUG    [2019-11-09 17:47:52,645] Description: Iran is acknowledging for the first time it has an open case before its Revolutionary Court over the 2007 disappearance of a former FBI agent on an unauthorized CIA mission to the country, renewing questions over what happened to him.  In a filing to the United Nations, Iran said the case over Robert Levinson was "on going," without elaborating.
+DEBUG    [2019-11-09 17:47:52,645] Links:
+[1]: https://news.yahoo.com/iran-says-case-open-ex-070518919.html(link)
+[2]: http://l2.yimg.com/uu/api/res/1.2/pRZD03lq5AKphL6noxcB1g--/YXBwaWQ9eXRhY2h5b247aD04Njt3PTEzMDs-/https://media.zenfs.com/en/ap.org/d83a715f4d3ea8efbb2129040f542c4a
+
+DEBUG    [2019-11-09 17:47:52,651] Title: Huge Hong Kong rally after student dies and lawmakers arrested
+DEBUG    [2019-11-09 17:47:52,651] Date: Fri, 08 Nov 2019 19:01:45 -0500
+DEBUG    [2019-11-09 17:47:52,651] Link: https://news.yahoo.com/hong-kong-pro-democracy-lawmakers-arrested-tensions-soar-060439204.html
+DEBUG    [2019-11-09 17:47:52,653] Description: Tens of thousands of Hong Kongers packed into a park Saturday night to mourn a student who died during recent clashes as police arrested a group of pro-democracy lawmakers, deepening the city's political crisis.  The international finance hub has been upended by five months of huge and increasingly violent pro-democracy protests, but Beijing has refused to give in to most of the movement's demands.  Tensions have soared since the death on Friday of Alex Chow, 22, who succumbed to head injuries sustained during a fall as police skirmished with demonstrators inside a car park last weekend.
+DEBUG    [2019-11-09 17:47:52,653] Links:
+[1]: https://news.yahoo.com/hong-kong-pro-democracy-lawmakers-arrested-tensions-soar-060439204.html(link)
+[2]: http://l1.yimg.com/uu/api/res/1.2/Cigo.wzpuf0tmOWdzOr1VA--/YXBwaWQ9eXRhY2h5b247aD04Njt3PTEzMDs-/http://media.zenfs.com/en_us/News/afp.com/574f0ffc2ba7be5934fcc6cdef859af0ee007aa3.jpg
+
```

Comment:
files containing logs are commonly are not checked in to git repository. better to remove it.

Reply:
Fixed